.github/workflows/test.yml

```diff
@@ -51,6 +52,16 @@ jobs:
       #  with:
       #    path: target
       #    key: ${{ runner.os }}-${{ steps.rustup.outputs.rustc_hash }}-target-${{ hashFiles('**/Cargo.lock') }}
+      - name: Setup Xcode env
+        if: matrix.os == 'macos-latest'
+        shell: bash
+        run: |
+          echo "PYO3_CROSS_LIB_DIR=/Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.8/lib" >> "${GITHUB_ENV}"
```

Comment:
`PYO3_CROSS_LIB_DIR` is kind of problematic for cross compiling from Inte/Apple Silicon to Apple Silicon/Intel, do we really need it in this situation?

Reply:
I don't know, and unfortunately I don't have a mac so I unfortunately can't check. That's something from the pyo3 build script, so if that's a problem it should be discussed in pyo3/pyo3

Reply:
Resolved by https://github.com/PyO3/pyo3/pull/1428